packages/mobile/src/transactions/feed/TransactionFeed.tsx

```diff
@@ -74,12 +80,58 @@ function useQueryTransactionFeed() {
   return { loading, error, transactions: result?.data?.tokenTransactionsV2.transactions }
 }
 
+function mapStandbyTransactionToTokenTransaction(tx: StandbyTransaction): TokenTransaction {
+  switch (tx.type) {
+    case TokenTransactionTypeV2.Exchange:
+      const exchangeTx = tx as ExchangeStandby
+      return {
+        __typename: 'TokenExchangeV2',
+        type: tx.type,
+        transactionHash: tx.hash || '',
+        timestamp: tx.timestamp,
+        block: '',
+        inAmount: {
+          value: exchangeTx.inValue,
+          tokenAddress: exchangeTx.inTokenAddress,
+        },
+        outAmount: {
+          value: exchangeTx.outValue,
+          tokenAddress: exchangeTx.outTokenAddress,
+        },
+        metadata: {},
+        fees: [],
+      }
+    default:
+      const transferTx = tx as TransferStandby
+      return {
+        __typename: 'TokenTransferV2',
+        type: tx.type,
+        transactionHash: tx.hash || '',
+        timestamp: tx.timestamp,
+        block: '',
+        address: transferTx.address,
+        amount: {
+          value: transferTx.value,
+          tokenAddress: transferTx.tokenAddress,
+        },
+        metadata: {},
+        fees: [],
+      }
+  }
+}
+
 function TransactionFeed() {
   const tokensInfo = useSelector(tokensByAddressSelector)
   const cachedTransactions = useSelector(transactionsSelector)
 
   const { loading, error, transactions } = useQueryTransactionFeed()
-  const tokenTransactions: TokenTransaction[] = transactions ?? cachedTransactions
+  const confirmedTokenTransactions: TokenTransaction[] = transactions ?? cachedTransactions
+
+  const standbyTransactions = useSelector(standbyTransactionsSelector).map((tx) =>
+    mapStandbyTransactionToTokenTransaction(tx)
+  )
+
+  const tokenTransactions = [...confirmedTokenTransactions, ...standbyTransactions]
```

Comment:
Shouldn't these by in the reverse order? ie. First the standby transactions and then the confirmed?

Also, when testing this the standby transaction doesn't show up as 'Confirming' what we used to have is an item that looked the same except that the subtitle said `Confirming...` (the `confirmingTransaction` translation text)

Reply:
Hmm, let me check this again